packages/ui/src/components/DatePicker.svelte

```diff
@@ -0,0 +1,255 @@
+<!--
+// Copyright © 2020 Anticrm Platform Contributors.
+// 
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import type { IntlString } from '@anticrm/platform'
+  import Label from './Label.svelte'
+  import PopupMenu from './PopupMenu.svelte'
+  import Calendar from './icons/Calendar.svelte'
+  import Close from './icons/Close.svelte'
+  import Back from './icons/Back.svelte'
+  import Forward from './icons/Forward.svelte'
+
+  export let title: IntlString
+  export let selected: Date = new Date(Date.now())
+  export let vAlign: 'top' | 'middle' | 'bottom' = 'bottom'
+  export let hAlign: 'left' | 'center' | 'right' = 'left'
+  export let margin: number = 16
+
+  let pressed: boolean = false
+  let view: Date = selected
+  const months: Array<string> = [
+    'Junary',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December'
+  ]
+  let monthYear: string
+  let days: Array<number>
+
+  const daysInMonth = (date: Date): number => {
+    return 33 - new Date(date.getFullYear(), date.getMonth(), 33).getDate()
+  }
+
+  $: {
+    monthYear = months[view.getMonth()] + ' ' + view.getFullYear()
+    days = []
+    for (let i = 1; i <= daysInMonth(view); i++) {
+      days.push(new Date(view.getFullYear(), view.getMonth(), i).getDay())
+    }
+    console.log('View:', view)
+    console.log('Selected:', selected)
+  }
+</script>
+
+<div class="userBox">
+  <PopupMenu {vAlign} {hAlign} {margin} bind:show={pressed} bind:title showHeader>
+    <button
+      slot="trigger"
+      class="btn"
+      class:selected={pressed}
+      on:click={(event) => {
```

Comment:
```suggestion
      on:click|prevent={(event) => {
```
Please check for svelte docs, I'm may be wrong with syntax.